packages/android_intent/example/pubspec.yaml

```diff
@@ -0,0 +1,53 @@
+name: android_intent_example
+description: Demonstrates how to use the android_intent plugin.
+
+dependencies:
+  flutter:
```

Comment:
I did do that where it made sense. I left comments like this because it accurately reflects what this is.